lib/src/map/camera/camera.dart

```diff
@@ -274,38 +272,39 @@ class MapCamera {
       final scale = getZoomScale(this.zoom, zoom);
       halfSize = size / (scale * 2);
     }
-    final pixelCenter = project(center, zoom).floor().toDoublePoint();
-    return Bounds(pixelCenter - halfSize, pixelCenter + halfSize);
+    final pixelCenter = project(center, zoom).floor();
+    return Bounds(
+        (pixelCenter - halfSize).toPoint(), (pixelCenter + halfSize).toPoint());
   }
 
   /// This will convert a latLng to a position that we could use with a widget
   /// outside of FlutterMap layer space. Eg using a Positioned Widget.
-  Point<double> latLngToScreenPoint(LatLng latLng) {
+  Offset latLngToScreenOffset(LatLng latLng) {
     final nonRotatedPixelOrigin = project(center, zoom) - nonRotatedSize / 2.0;
 
     var point = crs.latLngToPoint(latLng, zoom);
 
     final mapCenter = crs.latLngToPoint(center, zoom);
 
     if (rotation != 0.0) {
-      point = rotatePoint(mapCenter, point, counterRotation: false);
+      point = rotateOffset(mapCenter, point, counterRotation: false);
     }
 
     return point - nonRotatedPixelOrigin;
   }
 
   /// Calculate the [LatLng] coordinates for a [localPoint].
-  LatLng pointToLatLng(Point localPoint) {
-    final localPointCenterDistance = Point(
-      (nonRotatedSize.x / 2) - localPoint.x,
-      (nonRotatedSize.y / 2) - localPoint.y,
+  LatLng pointToLatLng(Offset localPoint) {
```

Comment:
Consistency, rename?

Now or in the future, we may also want to better document these or rename them to differentiate them from the CRS/projection level, which can be a confusing difference.